### PR TITLE
Replace Habit Event Detail/Edit/Delete fragment with an Activity

### PR DESCRIPTION
**Pull request by johnzhang27**
This PR is related to #96 and #122
* Implemented a new activity for Habit Event detail view and habit event edit/delete functionality.
* After discussion, we decided to disable change HabitID of a Habit Event.

### CODE REVIEW

File: code/HabitTracker/app/src/main/java/com/example/habittracker/activities/eventlist/EventListActivity.java [hunk 1]
```diff
@@ -121,62 +180,35 @@ public void onCallbackSuccess(ArrayList<Habit> habitList) {
...
+                                    eventDataList.add(doc);
                                 }
 
                             }
                             eventAdapter.notifyDataSetChanged(); // Notifying the adapter to render any new data fetched
                             //from the cloud
                         }
+
+                        @Override
+                        public void onCallbackFailed() {
```
Comment (PaoVal) [inline code/HabitTracker/app/src/main/java/com/example/habittracker/activities/eventlist/EventListActivity.java:209]: Should we have an error message pop up or something?

File: code/HabitTracker/app/src/main/java/com/example/habittracker/utils/CustomDatePicker.java [hunk 2]
```diff
@@ -31,7 +33,7 @@ public CustomDatePicker(Context context, View view, int et_id) {
         this.edittext = (EditText) view.findViewById(et_id);
         myCalendar = Calendar.getInstance();
         DatePickerDialog.OnDateSetListener theDP = this;
-        updateLabel(new Date());
+//        updateLabel(new Date());
```
Comment (PaoVal) [inline code/HabitTracker/app/src/main/java/com/example/habittracker/utils/CustomDatePicker.java:36]: This is ok. you can delete :)

File: code/HabitTracker/app/src/main/java/com/example/habittracker/utils/CustomDatePicker.java [hunk 3]
```diff
@@ -61,6 +86,7 @@ public void onDateSet(android.widget.DatePicker view, int year, int month, int d
         myCalendar.set(Calendar.YEAR, year);
         myCalendar.set(Calendar.MONTH, month);
         myCalendar.set(Calendar.DAY_OF_MONTH, dayOfMonth);
+//        updateLabel();
```
Comment (PaoVal) [inline code/HabitTracker/app/src/main/java/com/example/habittracker/utils/CustomDatePicker.java:89]: oh. I do need this though. I didn't think you had commented this out.... How else do you display the current date as a text to let the user know what they've chosen is selected?

File: code/HabitTracker/app/src/main/java/com/example/habittracker/activities/eventlist/EventListActivity.java (hunk 1 above)
Comment (johnzhang27) [reply code/HabitTracker/app/src/main/java/com/example/habittracker/activities/eventlist/EventListActivity.java:209]: Leave for later.

File: code/HabitTracker/app/src/main/java/com/example/habittracker/utils/CustomDatePicker.java (hunk 3 above)
Comment (johnzhang27) [reply code/HabitTracker/app/src/main/java/com/example/habittracker/utils/CustomDatePicker.java:89]: Line 90 should display current date for you:
updateLabel(myCalendar.getTime());

**Pull request merged**